src/modules/person/api/person.frontend.controller.ts

```diff
@@ -62,12 +62,20 @@ export class PersonFrontendController {
             organisationIDs = undefined;
         }
 
+        if (queryParams.organisationID) {
+            organisationIDs =
+                !organisationIDs || organisationIDs.includes(queryParams.organisationID)
```

Comment:
Wenn die organisationIDs auf die der Nutzer berechtigt ist (permissions.getOrgIdsWithSystemrecht) die angefragte organisationId nicht enthält, wird ein leeres Array assigned in weitergemacht. Aus meiner Sicht sollte hier ein NOT_AUTHORIZED geworfen werden, oder ist das bewusst so gemacht?

Desweiteren: wieso wird bei !organisationIDs weitergamcht. Auch hier ist doch der Nutzer eigendlich nicht berechtigt irgendwas zu sehen...?

Reply:
This was done deliberately because I thought that the frontend would be the one accessing this link, so it was not necessary. However, I will adjust it.

Reply:
User Frontend ist fast immer der Client der die Api nutzt. Dennoch kann ich Ohne Backend-Authorisierung als Nutzer einfach meinen Token nehmen und dann via e.g. Postman / Insomnia auf Resourcen zugreifen auf die ich eigendlich keinen Zugriff haben sollte. Bei unsensiblen GET Daten ist das natürlich weitaus weniger schlimm als bei sensiblen POST Resourcen...

Reply:
Das !organisationIDs ist rendundand, sofern du den Error weiter oben wirfst. Denn wenn dann der Fall !organisationIDs auftritt, durch z.B länge = 0, kann diese Stell garnicht mehr erreicht werden